Tests/test_libtiff_file_position.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import pytest
+
+from PIL import Image
+
+
+@pytest.mark.parametrize(
+    "test_file",
+    [
+        "Tests/images/old-style-jpeg-compression-no-samplesperpixel.tif",
+        "Tests/images/old-style-jpeg-compression.tif",
+    ],
```

Comment:
Is there anything different between these two files that means that one might conceivably fail where the other wouldn't? Or are you just testing two to be thorough?